shared/lib_battery_voltage.cpp

```diff
@@ -542,7 +540,7 @@ double voltage_vanadium_redox_t::calculate_current_for_target_w(double P_watts,
                                                                 this, _1, _2);
 
     double x[1], resid[1];
-    x[0] = solver_power / state->cell_voltage;
+    x[0] = solver_power / state->cell_voltage / 10;
```

Comment:
Given that it's in here twice, should 10 be a member variable of lib_battery_voltage?

Reply:
I don't think so because it's not inherent... it didn't have to be the same here and in the previous function I think